src/checks/stylelint.ts

```diff
@@ -0,0 +1,40 @@
+import { execaCommand } from 'execa';
+
+export const title = 'Stylelint';
+
+export default async function stylelintCheck() {
+  try {
+    await execaCommand(
+      'pnpm stylelint **/*.{css,scss,less,js,tsx} --max-warnings 0',
+    );
+  } catch (error) {
+    const { stdout } = error as { stdout: string };
+    const lines = stdout.split('\n');
+
+    // If no Stylelint problems detected, throw the error
+    if (
+      !/^\d+ problems? \(\d+ errors?, \d+ warnings\)$/.test(
+        lines[lines.length - 2]!,
+      )
+    ) {
+      throw error;
+    }
+
+    throw new Error(
+      `Stylelint problems found in the following files:
+
+      ${lines
```

Comment:
wonder if you'll need `normalizeNewline()` and to use `stdout` for Windows here like in the ESLint check:

https://github.com/upleveled/preflight/blob/c78424c54a81be5b97ecbf4073ccb73335d61239/src/checks/eslint.ts#L20

or maybe not, because you're not splitting

Reply:
https://github.com/upleveled/preflight/pull/377/commits/f0dbdb404c6b73fdfdc25cc8870d21dad9c7fef7